js/main.js

```diff
@@ -147,3 +153,38 @@ $unorderedList.addEventListener('click', function (event) {
     $headingTitle.textContent = 'Edit Entry';
   }
 });
+
+const $modal = document.querySelector('.modal');
+const $overlay = document.querySelector('.overlay');
+
+$deleteButton.addEventListener('click', function () {
+  $modal.classList.remove('hidden');
+  $overlay.classList.remove('hidden');
+});
+
+const $cancel = document.querySelector('.cancel');
+
+$cancel.addEventListener('click', function () {
+  $modal.classList.add('hidden');
+  $overlay.classList.add('hidden');
+});
+
+const $confirm = document.querySelector('.confirm');
+
+$confirm.addEventListener('click', function () {
+  const tempArray = [];
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].entryId !== $clickedElement) {
+      tempArray.push(data.entries[i]);
+    }
+  }
+  data.entries = tempArray;
```

Comment:
Works, but don't think the temp array is really necessary. You can just find the entry you need to remove and remove it with `splice`

Reply:
Thanks Shawn! Updated code and re-pushed for future reference!